bedrock/careers/tests/test_views.py

```diff
@@ -15,7 +15,7 @@
 
 class PositionTests(TestCase):
     def test_context(self):
-        response = self.client.get(reverse("careers.listings"), follow=True)
+        response = self.client.get(reverse("careers.listings"), follow=True, HTTP_ACCEPT_LANGUAGE="en")
```

Comment:
Side-thought: to avoid the locale header boilerplate, would it be worth using a [custom Client class](https://docs.djangoproject.com/en/3.2/topics/testing/tools/#customizing-the-test-client)? It looks like we could make it a [default header](https://github.com/django/django/blob/db588d4f0edde5095f8975be1da4e693268b86b3/django/test/client.py#L383)

Reply:
Possibly? But I think it's safe to assume an accept-language header for tests since that's what browsers will be doing. And we could override it explicitly to test the header-less flow. I can look into that.